kotlin/src/main/com/looker/rtl/Constants.kt

```diff
@@ -23,6 +28,22 @@ class DelimArray<T> : Array<T>() {
 }
  */
 
+fun isTrue(value: String?) : Boolean {
+    val low = value?.toLowerCase()
+    return low === "true" || low === "1" || low === "t" || low === "y" || low === "yes"
```

Comment:
I think the use of referential equality here may be a bit dangerous. If we're the ones instantiating the strings they should point to the same object, but if we're receiving this string from elsewhere, such as over the wire, they'll be different. This, for example, will return false:
```
"a" === buildString { "a" } 
```

Unless we absolutely want to ensure that these are the same object I'd relax this to `==`